projects/plugins/jetpack/modules/markdown/class-wpcom-markdown.php

```diff
@@ -128,7 +155,7 @@ public function load_markdown_for_posts() {
 		add_filter( 'edit_post_content', array( $this, 'edit_post_content' ), 10, 2 );
 		add_filter( 'edit_post_content_filtered', array( $this, 'edit_post_content_filtered' ), 10, 2 );
 		add_action( 'wp_restore_post_revision', array( $this, 'wp_restore_post_revision' ), 10, 2 );
-		add_filter( '_wp_post_revision_fields', array( $this, '_wp_post_revision_fields' ) );
+		add_filter( 'wp_post_revision_fields', array( $this, 'wp_post_revision_fields' ) );
```

Comment:
I don't know that we can change the filter name here, that's the way the filter is named in Core:
https://developer.wordpress.org/reference/hooks/_wp_post_revision_fields/

```suggestion
		add_filter( '_wp_post_revision_fields', array( $this, 'wp_post_revision_fields' ) );
```